src/coreclr/jit/hwintrinsiccodegenarm64.cpp

```diff
@@ -2052,6 +2052,38 @@ void CodeGen::genHWIntrinsic(GenTreeHWIntrinsic* node)
                 GetEmitter()->emitIns_R_R(ins, emitSize, targetReg, op1Reg, opt);
                 break;
 
+            case NI_Sve_Scatter:
+            {
+                if (!varTypeIsSIMD(intrin.op2->gtType))
+                {
+                    assert(intrin.numOperands == 4);
```

Comment:
can you add a comment of signature here, just as you have done on line 2080 below?

Reply:
Done